static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -77,25 +76,25 @@ const SearchDropdown = ({
     )}
 
     <DropdownFooter>
-      <ShortcutsRow>
+      <ButtonBar gap={1}>
         {runShortcut &&
-          visibleShortcuts?.map(shortcut => {
-            return (
-              <ShortcutButtonContainer
-                key={shortcut.text}
-                onClick={() => runShortcut(shortcut)}
-              >
-                <HotkeyGlyphWrapper>
-                  <HotkeysLabel
-                    value={shortcut.hotkeys?.display ?? shortcut.hotkeys?.actual ?? []}
-                  />
-                </HotkeyGlyphWrapper>
-                <IconWrapper>{shortcut.icon}</IconWrapper>
-                <HotkeyTitle>{shortcut.text}</HotkeyTitle>
-              </ShortcutButtonContainer>
-            );
-          })}
-      </ShortcutsRow>
+          visibleShortcuts?.map(shortcut => (
+            <Button
+              borderless
+              size="xs"
+              key={shortcut.text}
+              onClick={() => runShortcut(shortcut)}
+            >
+              <HotkeyGlyphWrapper>
+                <HotkeysLabel
+                  value={shortcut.hotkeys?.display ?? shortcut.hotkeys?.actual ?? []}
+                />
+              </HotkeyGlyphWrapper>
+              <IconWrapper>{shortcut.icon}</IconWrapper>
+              {shortcut.text}
+            </Button>
```

Comment:
**Before:**
<img width="238" alt="image" src="https://user-images.githubusercontent.com/44172267/187780877-2a069d6b-9a52-48a8-9e6d-22020a5169db.png">

**After:**
<img width="250" alt="image" src="https://user-images.githubusercontent.com/44172267/187780846-b05bed1b-a49b-414d-8986-cf3065935638.png">

--> Let's restyle these buttons to make them look like the before screenshot